COMPONENTS_DOCUMENTATION_GUIDELINES.md

```diff
@@ -0,0 +1,110 @@
+          <StoryDescription description="Large" vertical align={Flex.align.START}>
+              <Button size={Button.sizes.LARGE}>Large</Button>
+          </StoryDescription>
+          <StoryDescription description="Medium" vertical align={Flex.align.START}>
+              <Button size={Button.sizes.MEDIUM}>Medium</Button>
+          </StoryDescription>
+          <StoryDescription description="Small" vertical align={Flex.align.START}>
+              <Button size={Button.sizes.SMALL}>Small</Button>
+          </StoryDescription>
+      </Flex>
+  </Story>
+</Canvas>
+```
+Do this:
+```js
+<Canvas>
+    <Story name="Sizes">
+        <Button size={Button.sizes.LARGE}>Large</Button>
+        <Button size={Button.sizes.MEDIUM}>Medium</Button>
+        <Button size={Button.sizes.SMALL}>Small</Button>
+    </Story>
+</Canvas>
+```
+
+4. When laying out components, utilize Vibe's Flex component instead of implementing your own flex layout using CSS. Refer to the code example below for clarification. **Please use Flex only when displaying multiple examples** in a story and only if the default story layout and gaps do not match the documentation design without using Flex as a wrapper.
+```js
+<Flex direction={Flex.directions.COLUMN} gap={Flex.gaps.MEDIUM}>
+  <Steps type={Steps.types.NUMBERS} steps={steps} activeStepIndex={2} />
+  <Steps type={Steps.types.GALLERY} steps={steps} activeStepIndex={2} />
+</Flex>
+```
+
+4. If you need to add titles to the component states displayed in a story, please use the VisualDescription component, as shown in the code example below:
+```js
+<Flex direction={Flex.directions.ROW}>
+    <StoryDescription description="Xs">
+      <Loader size={Loader.sizes.XS} />
+    </StoryDescription>
+    <StoryDescription description="Small">
+      <Loader size={Loader.sizes.SMALL} />
+    </StoryDescription>
+</Flex>
+```
+6. For better story readability, it's best to use inline styles instead of class names when applying styles that are specific to the story itself. This avoids confusion for readers who might question the purpose of the class names in achieving the displayed result. Inline styles make the styles visible within the story, without the need for external CSS files or class definitions.
+```js
+<Canvas>
+  <Story name="Stretched">
+    <div style={{ width: "100%" }}>
+      <TabList tabType="stretched">
+        <Tab>First</Tab>
+        <Tab>Second</Tab>
+      </TabList>
+    </div>
+  </Story>
+</Canvas>
+```
+7. To keep our storybook code examples clear and simple, avoid using internal components (components which not exported to external users) in stories. Code duplication is acceptable if it means excluding internal components. This prevents confusion among developers using our components API. Frequent reliance on internal components for documentation suggests that the API lacks easy solutions for common use cases.  For example, instead of writing this:
```

Comment:
```suggestion
7. Avoid documenting wrapped components that are not exported to external users, as they are not available, and create an un-copyable example. If such cases, code duplication **is acceptable**. 

Instead of writing this:
```